modules/extensions-support/src/test/java/org/mule/runtime/module/extension/internal/runtime/exception/ExceptionHandlerManagerTestCase.java

```diff
@@ -82,15 +81,6 @@ public void processAndEnrich() {
     assertThat(throwable.getMessage(), is(ERROR_MESSAGE));
   }
 
-  @Test
-  public void handleConnectionException() {
```

Comment:
why is this test removed?

Reply:
After my changes, ExceptionHandlerManager does no longer handle a ConnectionException in a special way, so I removed the unit test that was verifying that.